package.json

```diff
@@ -2,29 +2,27 @@
   "name": "algorithmic-excercises",
   "version": "0.1.0",
   "description": "Algorithmic excercises",
-  "main": "index.js",
+  "main": "FIXME",
   "scripts": {
     "python:pip:install:upgrade": "python -m pip install --upgrade --requirement requirements.upgrade.txt",
     "python:pipx:install": "bash scripts/pipx/install.ba.sh",
     "python:flake8": "python -m flake8 src --benchmark --statistics",
-    "python:pyright": "python -m pyright src --warnings",
-    "python:pypyr": "python -m pypyr",
-    "setup": "npm run python:pip:install:upgrade && npm run python:pipx:install",
     "prettier": "prettier .",
     "prettier:check": "npm run prettier -- --check",
     "prettier:write": "npm run prettier -- --write",
-    "lint": "npm run python:pypyr -- lint",
-    "style": "npm run python:pypyr -- style",
-    "test": "npm run python:pypyr -- test",
     "poetry:shell": "poetry shell",
     "poetry:install": "poetry install",
     "_poetry:export": "poetry export --no-interaction --format requirements.txt",
     "poetry:export:dev": "npm run _poetry:export -- --output requirements.dev.txt --with dev",
     "poetry:export:prd": "npm run _poetry:export -- --output requirements.prd.txt",
     "poetry:export": "npm run poetry:export:dev && npm run poetry:export:prd",
     "pre-commit:install": "pre-commit install",
-    "dev": "npm run poetry:shell",
-    "postinstall": "npm run poetry:install && npm run pre-commit:install"
+    "setup": "npm run python:pip:install:upgrade && npm run python:pipx:install",
+    "setup:poetry": "npm run poetry:install && npm run pre-commit:install",
+    "lint": "npm run prettier:check",
+    "style": "npm run prettier:write",
+    "poststyle": "npm run lint",
+    "test": "exit 0"
```

Comment:
I've been trying to use `npm scripts` as a way to NOT  have to write custom `scripts/`

But I've learned to appreciate the difference between
- `npm scripts` npm only stuff (i.e. `prettier`)
- `pypyr pipelines` for python only stuff (i.e. `ruff` & `pytest`)
- `bash scripts` for things more complicated that would make anything afromentioned horribly-complicated